components/Logo/svg/BlueLogo.tsx

```diff
@@ -0,0 +1,51 @@
+const BlueLogo = () => {
+  return (
+    <svg
+      aria-label="Juguetear"
+      data-testid="blue-logo"
+      width="315"
+      height="201"
+      viewBox="0 0 315 201"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <path
+        d="M241.101 83.3142C244.886 83.3142 247.955 80.1698 247.955 76.2909C247.955 72.412 244.886 69.2676 241.101 69.2676C237.315 69.2676 234.246 72.412 234.246 76.2909C234.246 80.1698 237.315 83.3142 241.101 83.3142Z"
+        fill="#00487C"
```

Comment:
Para todos los fills y strokes es mejor usar clases en vez del color directamente. De esa manera, si un color se actualiza, se actualiza en todos lados. En este caso podés agregar este azul al archivo tailwind.config.js (ej `darkblue: #00487C`) y despues hacer `className="fill-darkblue"`
https://tailwindcss.com/docs/fill#setting-the-fill-color
https://tailwindcss.com/docs/stroke#setting-the-stroke-color

Reply:
Gracias por la info!